plumbing/proofs/proofs.go

```diff
@@ -0,0 +1,35 @@
+package proofs
+
+import (
+	"github.com/filecoin-project/go-filecoin/proofs"
+	"github.com/filecoin-project/go-filecoin/proofs/sectorbuilder"
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+// Provides the sector builder for proofs.
+type sectorBuilderer interface {
+	SectorBuilder() sectorbuilder.SectorBuilder
```

Comment:
Since it provides the sector builder for proofs, can we call it sectorBuilderProvider instead?